x-pack/test/functional/apps/canvas/reports.ts

```diff
@@ -226,129 +195,152 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
-        /BitsPerComponent 8
-        /Filter /FlateDecode
-        /ColorSpace /DeviceGray
-        /Decode [0 1]
-        /Length 12
-        >>
-        "
+          "
+          endobj
+          10 0 obj
+          <<
+          /Type /ExtGState
+          /ca 1
+          /CA 1
+          >>
+          endobj
+          9 0 obj
+          <<
+          /Type /Page
+          /Parent 1 0 R
+          /MediaBox [0 0 8 8]
+          /Contents 7 0 R
+          /Resources 8 0 R
+          >>
+          endobj
+          8 0 obj
+          <<
+          /ProcSet [/PDF /Text /ImageB /ImageC /ImageI]
+          /ExtGState <<
+          /Gs1 10 0 R
+          >>
+          /XObject <<
+          /I1 5 0 R
+          >>
+          >>
+          endobj
+          7 0 obj
+          <<
+          /Length 47
+          /Filter /FlateDecode
+          >>
+          "
         `);
 
         // PDF Info
         expectSnapshot(info).toMatchInline(`
-        "
-        endobj
-        5 0 obj
-        <<
-        /Type /XObject
-        /Subtype /Image
-        /BitsPerComponent 8
-        /Width 16
-        /Height 16
-        /Filter /FlateDecode
-        /ColorSpace /DeviceRGB
-        /SMask 14 0 R
-        /Length 17
-        >>
-        "
+          "
+          endobj
+          12 0 obj
+          (pdfmake)
+          endobj
+          13 0 obj
+          (pdfmake)
+          endobj
+          14 0 obj
+          (D:20240805093833Z)
+          endobj
+          11 0 obj
+          <<
+          /Producer 12 0 R
+          /Creator 13 0 R
+          /CreationDate 14 0 R
+          >>
+          endobj
+          4 0 obj
+          <<
+          >>
+          endobj
+          3 0 obj
+          <<
+          /Type /Catalog
+          /Pages 1 0 R
+          /Names 2 0 R
+          >>
+          endobj
+          1 0 obj
+          <<
+          /Type /Pages
+          /Count 1
+          /Kids [9 0 R]
+          >>
+          endobj
+          2 0 obj
+          <<
+          /Dests <<
+            /Names [
+          ]
+          >>
+          >>
+          endobj
+          xref
+          0 15
+          0000000000 65535 f 
+          0000000939 00000 n 
+          0000000996 00000 n 
+          0000000877 00000 n 
+          0000000856 00000 n 
+          0000000092 00000 n 
+          0000000015 00000 n 
+          0000000574 00000 n 
+          0000000453 00000 n 
+          0000000353 00000 n 
+          0000000302 00000 n 
+          0000000780 00000 n 
+          0000000692 00000 n 
+          0000000718 00000 n 
+          0000000744 00000 n 
+          trailer
+          <<
+          /Size 15
+          /Root 3 0 R
+          /Info 11 0 R
+          /ID [<b7f7680c53438aa6171c4790e359b0eb> <b7f7680c53438aa6171c4790e359b0eb>]
+          >>
+          startxref
+          1043
+          %%EOF
+          "
         `);
 
-        expect(res.get('content-length')).to.be('1600');
+        expect(res.get('content-length')).to.be('1498');
```

Comment:
Tests snapshot were updated to unblock this upgrade it's unclear why there's suddenly a change in the result received and the test expectation, regardless I did manually verify the PDF generated previously[^1] compared to the PDF that gets generated going forward from this PR[^2] for this particular test and they both compare to be the same visually. That being said, I spoke with @tsullivan and we both agree that this method for catching visual regression doesn't seem reliable, so some effort will be made into https://github.com/elastic/kibana/issues/117814 to find a solution that's much more reliable. 

footnotes:
[^1]: [The Very Cool Workpad for PDF Tests (1).pdf](https://github.com/user-attachments/files/16494107/The.Very.Cool.Workpad.for.PDF.Tests.1.pdf)
[^2]: [The Very Cool Workpad for PDF Tests.pdf](https://github.com/user-attachments/files/16494114/The.Very.Cool.Workpad.for.PDF.Tests.pdf)